cpu/fe310/cpu.c

```diff
@@ -188,10 +199,20 @@ void handle_trap(unsigned int mcause)
         }
     }
     else {
+#ifdef DEVELHELP
+        printf("Unhandled trap:\n");
+        printf("  mcause: 0x%08x\n", mcause);
+        printf("  mepc:   0x%08x\n", mepc);
+        printf("  mtval:  0x%08x\n", mtval);
+#endif
         /* Unknown trap */
         core_panic(PANIC_GENERAL_ERROR, "Unhandled trap");
     }
 
+    /* Check if context change was requested */
+    if( sched_context_switch_request )
+        sched_run();
```

Comment:
RIOT coding style would be
```C
    if (sched_context_switch_request) {
        sched_run();
    }
```

Reply:
Fixed with commit

Reply:
Not quite…
The space should be after the `if`, not after the `(`.

Reply:
Fixed for real this time.